src/contentscript/overlay/root/pages/Settings/Developer/Developer.tsx

```diff
@@ -2,13 +2,13 @@ import { initBGFunctions } from 'chrome-extension-message-wrapper'
 import cn from 'classnames'
 import React, { FC, useEffect, useRef, useState } from 'react'
 import { browser } from 'webextension-polyfill-ts'
-import ModuleInfo from '../../../../../background/models/moduleInfo'
-import VersionInfo from '../../../../../background/models/versionInfo'
-import { groupBy } from '../../../../../common/helpers'
-import { isValidUrl } from '../../../../../popup/helpers'
-import { DevModule } from '../../components/DevModulesList'
-import { Localhost } from '../../components/Localhost'
-import { Registry } from '../../components/Registery'
+import ModuleInfo from '../../../../../../background/models/moduleInfo'
+import VersionInfo from '../../../../../../background/models/versionInfo'
```

Comment:
Write in the technical debt. Looks like shared types, but it's not just a type, it's model classes... Do not refactor now, write to the debt